core/src/test/java/org/hibernate/ogm/backendtck/associations/manytoone/ManyToOneTest.java

```diff
@@ -42,8 +42,8 @@ public void testUnidirectionalManyToOne() throws Exception {
 		session.persist( emmanuel );
 		session.persist( jerome );
 		session.flush();
-		assertThat( getNumberOfEntities( sessions ) ).isEqualTo( 3 );
-		assertThat( getNumberOfAssociations( sessions ) ).isEqualTo( expectedAssociations() );
+		assertThat( getNumberOfEntities( session ) ).isEqualTo( 3 );
```

Comment:
How comes this has changed now? It seems to actually use only the SF eventually anyways?